cirq/aqt/aqt_device.py

```diff
@@ -12,9 +12,9 @@
 """
 import json
 from typing import Union, Tuple, List, Sequence
-from cirq import Circuit, ops, LineQubit, study, IonDevice, DensityMatrixSimulator, NoiseModel
+from cirq import Circuit, ops, LineQubit, study, IonDevice, DensityMatrixSimulator, NoiseModel, NO_NOISE
```

Comment:
Import packages, like `ops` or `circuits` or `noise`, instead of individual values.